.github/workflows/ci.yml

```diff
@@ -268,6 +267,7 @@ jobs:
     - name: Build Docs
       run: |
         docs/generate.sh --persist
+        rm -r docs/out/wk-repo
```

Comment:
This folder was still used for the Changelog & Migration Guide. I just symlinked the docs to `webknossos` do have a better url `docs.webknossos.org/webknossos` instead of `docs.webknossos.org/wk-repo/docs`.

Reply:
hmm. it doesn't make sense to copy the whole repo to s3 every time. can we just copy/keep some files?

Reply:
Sure, how about:
```
* docs
  * wk-repo
  * src
    * webknossos --> ../wk-repo/docs
```
And we simply link the changelog and migration guide also in the wk repo from top-level into docs. That way only the files we need part of the src folder. Should I adapt it that way?

Reply:
yes please